### PR TITLE
Fix small errors in analysis-time tutorials

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
This PR is to correct some small errors in the analysis-time tutorials, for example spelling mistakes and correcting the referencing. 

Again this is a draft as I could not run the docs locally, so do not know if it is all corrected properly.

### CODE REVIEW

**Comment by registerrier** [comment]
Can you rebase now that #5020 has been merged?

**Comment by Astro-Kirsty** [comment]
I have rebased

File: examples/tutorials/analysis-time/Variability_estimation.py [hunk 1]
```diff
@@ -54,10 +63,13 @@
 ######################################################################
 # Methods to characterize variability
 # -----------------------------------
-#  Amplitude maximum variation, relative variability amplitude and variability amplitude
+#
+# The 3 methods shown here are: amplitude maximum variation, relative variability amplitude
```
Comment (registerrier) [inline examples/tutorials/analysis-time/Variability_estimation.py:67]: YOu could even use bullet points.

**Review by registerrier — DISMISSED** [review]
Thanks @Astro-Kirsty . This looks good. Are the links you added correctly rendered?

File: examples/tutorials/analysis-time/Variability_estimation.py (hunk 1 above)
Comment (Astro-Kirsty) [reply examples/tutorials/analysis-time/Variability_estimation.py:67]: Good idea!

**Comment by Astro-Kirsty** [comment]
All links are rendered correctly, I cannot show on my github documentation page because my other PR is shown there currently!

**Comment by registerrier** [comment]
CI fail is unrelated. Merging.

**Pull request merged**